.github/workflows/test-e2e.yml

```diff
@@ -52,8 +52,11 @@ jobs:
       - name: Install dependencies
         run: npm ci
       # install OS dependencies required by browsers on the GitHub runner
+      # to be sure that the browser is correctly installed: https://github.com/microsoft/playwright/issues/5801
       - name: Install the dependencies for ${{matrix.browser}} only
-        run: npx playwright install-deps ${{matrix.browser}}
+        run: |
+          npx playwright install ${{matrix.browser}}
```

Comment:
⚠️ if we download and install the browser here, we should skip browser installation during the `npm ci` as for bundle tests run in the build workflow